### PR TITLE
Master marketing bs tde

### PR DESCRIPTION
**Pull request by tde-banana-odoo**
Task: https://www.odoo.com/web#id=1872198&action=333&active_id=965&model=project.task&view_type=form&menu_id=4720

### CODE REVIEW

**Comment by Elkasitu** [comment]
does the `bs` stand for `bacon strips`? 🤣

**Comment by tde-banana-odoo** [comment]
... Yeah :roll_eyes:

File: addons/mass_mailing_crm/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Mass mailing on attendees',
```
Comment (tde-banana-odoo) [inline addons/mass_mailing_crm/__manifest__.py]: Todo: update this

File: addons/mass_mailing_crm/views/mass_mailing_views.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+<odoo>
+    <record id="mail_mass_mailing_view_form" model="ir.ui.view">
+        <field name="name">mail.mass_mailing.view.form.inherit.crm</field>
+        <field name="model">mail.mass_mailing</field>
+        <field name="inherit_id" ref="mass_mailing.view_mail_mass_mailing_form"/>
+        <field name="arch" type="xml">
+    		<xpath expr="//button[hasclass('o_mass_mailing_button')]" position="before">
+    			<field name="crm_lead_activated" invisible="1"/>
+    			<button name="action_redirect_to_leads"
+                    type="object"
```
Comment (tde-banana-odoo) [inline addons/mass_mailing_crm/views/mass_mailing_views.xml]: Todo: check indentation

File: addons/mass_mailing_sale/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Mass mailing on attendees',
```
Comment (tde-banana-odoo) [inline addons/mass_mailing_sale/__manifest__.py]: Todo: update this

File: addons/mass_mailing_sale/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    'category': 'Hidden',
+    'version': '1.0',
+    'summary': 'Add Leads/Opportunities info on mass mailing',
```
Comment (tde-banana-odoo) [inline addons/mass_mailing_sale/__manifest__.py]: Todo: update this

File: addons/mass_mailing_sale/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    'description':
+        """
+Mass mail leads / opportunities
```
Comment (tde-banana-odoo) [inline addons/mass_mailing_sale/__manifest__.py]: Todo: update this

**Pull request merged**